app/src/main/java/com/poupa/vinylmusicplayer/discog/AddSongAsyncTask.java

```diff
@@ -17,87 +10,52 @@
  * @author SC (soncaokim)
  */
 
-class AddSongAsyncTask extends AsyncTask<Song, Void, Boolean> {
+class AddSongAsyncTask extends AsyncTask<Song, Void, Integer> {
```

Comment:
The main changes in this class are:
- the counter of processed song (instead of doing +1 at each step, we may have +N
- move of UI bits (use of Snackbar) in a dedicated SnackBarUtil class